README.md

```diff
@@ -3,6 +3,13 @@
 This is a demo project for showing how to run unit tests with Cloud
 Container Builder.
 
+## Test Runners
+
+Currently we only have a test runner for phpunit at `php/test/runner`
+directory. We built it and push it to
+gcr.io/cloud-dpes/phpunit-test-runner. This image is publicly
```

Comment:
We should be able to make this repository visible to everyone. Currently, following that link results in "You don't have permission to view Container Registry".